kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -198,6 +197,26 @@ public String convert(boolean heatCoolEq, StringBuilder sb) {
         }
         translateSymbols(attributes, functionRules, impurities, overloads, sb);
 
+        // print syntax definition
+        int length = sb.length();
+        for (Tuple2<Sort, scala.collection.immutable.List<Production>> sort : iterable(module.bracketProductionsFor())) {
+            for (Production prod : iterable(sort._2())) {
+                translateSymbol(attributes, functionRules, impurities, overloads, prod.att().get("bracketLabel", KLabel.class), prod, sb);
+            }
+            for (Production prod : iterable(module.sortedProductions())) {
```

Comment:
Why is this `for` loop inside the above one? It does not seem to depend on `sort`, which, I guess, would mean that `genSubsortAxiom` below generates multiple identical axioms (assuming that the `bracketProductionsFor` can have more than one entry).

Reply:
good catch. I did not notice that.